SonarQube.TeamBuild.Integration/CoverageReportDownloader.cs

```diff
@@ -59,6 +60,10 @@ private void InternalDownloadReport(string tfsUri, string reportUrl, string repo
                         response.Content.CopyToAsync(fileStream).Wait();
                     }
                 }
+                else
+                {
+                    logger.LogError(Resources.PROC_ERROR_FailedToDownloadReportReason, response.StatusCode, response.ReasonPhrase);
```

Comment:
This SHOULD work. Unfortunately I could not get Fiddler to capture traffic generated by the post-processor, in order to manipulate it to send errors ...